components/infobox/wikis/leagueoflegends/infobox_league_custom.lua

```diff
@@ -0,0 +1,120 @@
+---
+-- @Liquipedia
+-- wiki=leagueofledends
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Template = require('Module:Template')
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.liquipediaTierHighlighted = CustomLeague.liquipediaTierHighlighted
+	league.appendLiquipediatierDisplay = CustomLeague.appendLiquipediatierDisplay
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(args.team_number or '')}
+	})
+	table.insert(widgets, Cell{
+		name = 'Players',
+		content = {args.participants_number}
+	})
+	return widgets
+end
+
+function CustomLeague:appendLiquipediatierDisplay()
+	if String.isEmpty(_args.pctier) and Logic.readBool(_args.riotpremier) then
+		return ' ' .. Template.safeExpand(mw.getCurrentFrame(), 'Riot/infobox')
+	end
+	return ''
+end
+
+function CustomLeague:liquipediaTierHighlighted()
+	return Logic.readBool(_args.riotpremier)
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.participantsnumber = args.player_number or args.team_number
+	lpdbData.extradata = {
+		individual = String.isNotEmpty(args.player_number) and 'true' or '',
```

Comment:
`individual={{#if:{{{individual|}}}{{{participants_number|}}}|true}}`
so should be
```suggestion
		individual = String.isNotEmpty(args.participants_number) or String.isNotEmpty(args.individual) and 'true' or '',
```